include/FockSpace/FockSpace.hpp

```diff
@@ -3,47 +3,57 @@
 
 
 #include "FockSpace/BaseFockSpace.hpp"
+
 #include <boost/numeric/conversion/converter.hpp>
 #include <boost/math/special_functions.hpp>
 
 
 namespace GQCG {
 
 
+/**
+ *  The full Fock space for given set of orbitals and number of electrons
+ *  where the ONV's and addresses are linked
```

Comment:
The ONVs and addresses are linked through a hashing function calculated with an addressing scheme.